### DISCUSSION TITLE
unable to open the browser

### DISCUSSION THREAD

**Discussion by Aampy** [akash-coded/mern#37] [Ideas]
Sir, I am unable to open the browser from vs code as run it on the live server 
![Screenshot (78)](https://user-images.githubusercontent.com/106874799/172331513-45eb074c-55ad-4ee4-b8ce-16d045357ab0.png)

**Comment by santu1667** [comment]
@Aampy   Search for live server and install it instead of Open. After installed right click on the html page and you will see option "open with Live server"

**Reply by Aampy** [reply]
Even that doesn't work
